### ISSUE TITLE
[RadListView] ListView background is now white on android (regression)

### ISSUE THREAD

**Issue by edusperoni** [NativeScript/nativescript-ui-feedback#1254]
> Please take a minute to read our [NativeScript Code of Conduct](https://github.com/NativeScript/codeofconduct) before proceeding with posting issues or discussing. The purpose of this guide is to make communication and cooperation within our forums a pleasure for you and the other members.

### Tell us about the problem
The RadListView background is no longer respected for item templates. Headers are ok.

### Which platform(s) does your issue occur on?
Android

### Please provide the following version numbers that your issue occurs with:
Issue only occurs with nativescript-ui-listview 7.1.0 and 7.0.5. Reverting to 7.0.4 fixes the issue
### Please tell us how to recreate the issue in as much detail as possible.
Add a Listview with `backgroundColor="blue"` and the following template:
```
<StackLayout backgroundColor="black" marginTop="10" height="100"></StackLayout>
```
There will be a white margin on top, instead of a blue one.

**Comment by VladimirAmiorkov** [comment]
Hi @edusperoni ,

Thank you for reporting this issue. We have already observed it and have found its source. It is being tested so you can expect and patch fix version very soon.

**Comment by AxelPAL** [comment]
@mpht, works perfectly on Android. Thank you!

**Comment by zbranzov** [comment]
The fix is released in nativescript-ui-listview@8.0.0
